includes/class-genesis-tabs-widget.php

```diff
@@ -0,0 +1,422 @@
+<?php
+/**
+ * Genesis Tabs Widget file.
+ *
+ * @package genesis-tabs
+ */
+
+/**
+ * Widget Class. Handles the widget form and output.
+ *
+ * @package Genesis_Tabs
+ * @since 0.9.0
+ */
+class Genesis_Tabs_Widget extends WP_Widget {
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		$widget_ops  = array(
+			'classname'   => 'ui-tabs',
+			'description' => __( 'Displays featured posts in Tabs', 'genesis' ),
+		);
+		$control_ops = array(
+			'width'   => 505,
+			'height'  => 350,
+			'id_base' => 'tabs',
+		);
+		parent::__construct( 'tabs', __( 'Featured Tabs', 'genesis' ), $widget_ops, $control_ops );
+	}
+
+	/**
+	 * Widget Output
+	 *
+	 * @param array $args Args.
+	 * @param array $instance Instance.
+	 */
+	public function widget( $args, $instance ) {
+		extract( $args );
+
+		/* defaults */
+		$instance = wp_parse_args(
+			(array) $instance,
+			array(
+				'title'           => '',
+				'posts_cat_1'     => '',
+				'posts_cat_2'     => '',
+				'posts_cat_3'     => '',
+				'posts_cat_4'     => '',
+				'posts_cat_5'     => '',
+				'posts_cat_6'     => '',
+				'posts_cat_7'     => '',
+				'posts_cat_8'     => '',
+				'posts_cat_9'     => '',
+				'posts_cat_10'    => '',
+				'show_image'      => 0,
+				'image_alignment' => '',
+				'image_size'      => '',
+				'show_title'      => 0,
+				'show_byline'     => 0,
+				'post_info'       => '[post_date] ' . __( 'By', 'genesis' ) . ' [post_author_posts_link] [post_comments]',
+				'show_content'    => 'excerpt',
+				'content_limit'   => '',
+				'more_text'       => __( '[Read More...]', 'genesis' ),
+			)
+		);
+
+		echo esc_attr( $before_widget );
+
+			/* Output Widget Title */
+		if ( ! empty( $instance['title'] ) ) {
+			echo esc_attr( $before_title . apply_filters( 'widget_title', $instance['title'], $instance, $this->id_base ) . $after_title );
+		}
+
+			/* Pull the chosen categories into an array */
+			$cats = array( $instance['posts_cat_1'], $instance['posts_cat_2'], $instance['posts_cat_3'], $instance['posts_cat_4'], $instance['posts_cat_5'], $instance['posts_cat_6'], $instance['posts_cat_7'], $instance['posts_cat_8'] );
+
+			/* Display the tab links */
+			echo '<ul class="ui-tabs-nav">';
+		foreach ( (array) $cats as $cat ) {
+			if ( $cat ) {
+				echo '<li><a href="#cat-' . esc_html( $cat ) . '">' . esc_html( get_cat_name( $cat ) ) . '</a></li>';
+			}
+		}
+			echo '</ul>';
+
+			/* Loop through all chosen categories */
+		foreach ( (array) $cats as $cat ) :
+
+			if ( ! $cat ) {
+				continue; /* skip iteration if $cat is empty */
+			}
+
+			/* Custom loop */
+			$tabbed_posts = new WP_Query(
+				array(
+					'cat'       => $cat,
+					'showposts' => 1,
+					'orderby'   => 'date',
+					'order'     => 'DESC',
+				)
+			);
+			if ( $tabbed_posts->have_posts() ) :
```

Comment:
I am not sure why phpcs didn't pick this up, but would you mind switching the clause enclosures to brackets?

**from** 
```
if ( something ) :
    do something
endif;
```

to
```
if ( something ) {
    do something
}
```